r-package/grf/R/input_utilities.R

```diff
@@ -187,28 +190,28 @@ create_train_matrices <- function(X,
   out <- list()
   i <- 0
   if (!is.null(outcome)) {
-    i <- i + 1
-    out[["outcome.index"]] <- ncol(X) + i
+    i <- i + NCOL(outcome)
```

Comment:
Note: `NCOL(x)` returns the length of x in case x is a vector, which is the desired behavior here unlike `ncol(x)` which would return `NULL`